### PR TITLE
Decouple filters and columns

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | See https://github.com/PrestaShop/prestafony-project/issues/62
| Type?         |  improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | Email logs, Logs and Sql Manager grids should work the same.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9442)

### CODE REVIEW

**Comment by sarjon** [comment]
@mickaelandrieu can you take a look?

File: src/PrestaShopBundle/Translation/TranslatorAwareTrait.php [hunk 1]
```diff
@@ -47,4 +47,18 @@ public function setTranslator(TranslatorInterface $translator)
...
+    /**
+     * Shortcut method to translate text
+     *
+     * @param string $id
+     * @param array $options
+     * @param string $domain
+     *
+     * @return string
+     */
+    protected function trans($id, array $options, $domain)
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Translation/TranslatorAwareTrait.php:60]: 👍

**Review by mickaelandrieu — DISMISSED** [review]
This need unit tests and update of grid docs but yes, I agree on decoupling filters to columns.

Good job @sarjon

**Comment by sarjon** [comment]
Updated Logs grid & unit tests. :+1:

**Review by mickaelandrieu — DISMISSED** [review]

**Comment by mickaelandrieu** [comment]
Hello @PrestaShop/prestashop-core-developers I'd like someone to review in order to learn a little bit about the Grid system please 👍 . Docs will come, but how can we validate it if we don't know how it works?

**Comment by mickaelandrieu** [comment]
Hello @sarjon, a little rebase here please so I can validate and merge this contribution.

**Comment by sarjon** [comment]
@mickaelandrieu done

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
good2me

**Comment by mickaelandrieu** [comment]
Thanks @sarjon !

**Pull request merged**